scripts/test-coverage.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+#
+# Generate test coverage reports.
+#
+# To use, point this at the files you want a coverage report on.  This opens a
+# browser with a report or point you at the report if it can't find a browser.
+#
+# You will need a few tools installed, but this script should be able to install
+# those for you using `rustup` or `cargo`.
+
+set -e
+
+tests=()
```

Comment:
Can you provide an example of how to use this script and how will the output look like? Maybe can attach a screenshot with the output?

Reply:
I ran:

```
$ ./scripts/test-coverage.sh src/helpers/buffers/unordered_receiver.rs
```

After building and running the tests in that file, it opened a browser to a coverage report for that file.  That shows a few assertions and even a few lines of real code that are not covered by those unit tests.

To Alex's point, `cargo llvm-cov` does much the same thing.  I've just automated it more.  I'm happy to keep this for my own use if others don't find it useful.

Reply:
If we can automate it to a point where we can see coverage on PRs, I would be happy